data-model/src/main/avro/RawSpan.avdl

```diff
@@ -18,6 +19,8 @@ protocol RawSpanProtocol {
     Event event;
 
     long received_time_millis = 0;
+
+    union { null, EventLogs } eventLogs = null;
```

Comment:
Its already been added into Event record, so we don't need it here in RawSpan.

Reply:
RawSpan is the structure written out by span-normalizer, so it is needed in that right